src/coreclr/vm/arm/ehhelpers.S

```diff
@@ -132,12 +132,14 @@ GenerateRedirectedStubWithFrame RedirectForThreadAbort, RedirectForThreadAbort2
         // On entry:
         //
         // R0 = throwable
-        // R1 = SP of the caller of the method/funclet containing the filter
+        // R1 = FP of main method
         // R2 = PC to invoke
         // R3 = address of the location where the SP of funclet's caller (i.e. this helper) should be saved.
         //
         // Save the SP of this function
         str sp, [r3]
+        // Restore frame pointer
+        mov r11, r1
```

Comment:
Do we need to push/pop r11 in the prolog/epilog?

Reply:
I will double check. There's preexisting mismatch between native and managed calling conventions. Thumb code emitted by C++ compiler uses `r7` register for frame pointer (due to limitation of the 2-byte Thumb PUSH instruction to access only r0-r7 registers).

Reply:
We should, I will push a fix (after the NativeAOT Outerloop CI run finishes). Good catch!